### ISSUE TITLE
/me error breaks chatcontrol

### ISSUE THREAD

**Issue by LogGits** [kangarko/ChatControl-Pro#1537]
```
!-----------------------------------------------------!
JavaScript placeholders will not function!
Your Java version/distribution lacks
the Nashorn library for JavaScript
placeholders. Unfortunately this plugin
relies heavily on Nashorn and cannot
continue. Please install Oracle Java 8.
Plugin is now disabled.
!-----------------------------------------------------!
[ChatControl]: Disabling ChatControl v8.5.4
ChatControl 8.5.4 encountered a NullPointerException! Please check your error.log and report this issue with the information in that file.
java.lang.NullPointerException
at org.mineacademy.chatcontrol.lib.fo.model.Lpt3.const(SourceFile:114)
at org.mineacademy.chatcontrol.final.nul. .L.const(SourceFile:72)
at org.mineacademy.chatcontrol.final.nul. .K.const(SourceFile:146)
at org.mineacademy.chatcontrol.final.nul. .K.const(SourceFile:128)
at org.mineacademy.chatcontrol.final.nul. .K.const(SourceFile:124)
at org.mineacademy.chatcontrol.final.nul. .class.const(SourceFile:34)
at org.mineacademy.chatcontrol.final.nul. .LPt8.execute(SourceFile:322)
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
at java.lang.Thread.run(Thread.java:748)
!-----------------------------------------------------!
Failed to execute command /me ¯\_(ツ)_/¯
!-----------------------------------------------------!
```
paper 1.15.2 build 67
Chc 8.5.4

**Comment by LogGits** [comment]
Apparently any /me command breaks it
```
!-----------------------------------------------------!
JavaScript placeholders will not function!
Your Java version/distribution lacks
the Nashorn library for JavaScript
placeholders. Unfortunately this plugin
relies heavily on Nashorn and cannot
continue. Please install Oracle Java 8.
Plugin is now disabled.
!-----------------------------------------------------!
[ChatControl]: Disabling ChatControl v8.5.4
ChatControl 8.5.4 encountered a NullPointerException! Please check your error.log and report this issue with the information in that file.
java.lang.NullPointerException
at org.mineacademy.chatcontrol.lib.fo.model.Lpt3.const(SourceFile:114)
at org.mineacademy.chatcontrol.final.nul. .L.const(SourceFile:72)
at org.mineacademy.chatcontrol.final.nul. .K.const(SourceFile:146)
at org.mineacademy.chatcontrol.final.nul. .K.const(SourceFile:128)
at org.mineacademy.chatcontrol.final.nul. .K.const(SourceFile:124)
at org.mineacademy.chatcontrol.final.nul. .class.const(SourceFile:34)
at org.mineacademy.chatcontrol.final.nul. .LPt8.execute(SourceFile:322)
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
at java.lang.Thread.run(Thread.java:748)
!-----------------------------------------------------!
Failed to execute command /me is blessed
!-----------------------------------------------------!
```

**Comment by lackoSK** [comment]
Hey,

Please **install Jave version 8** 

Thanks!

**Comment by lackoSK** [comment]
Hello,

This issue is being resolved in ticket #1531

I will close this one - Hope you understand.

Regards,
lackoSK
